x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -94,7 +94,7 @@ export const updateAlertStatusAction = async ({
     // TODO: Only delete those that were successfully updated from updatedRules
     setEventsDeleted({ eventIds: alertIds, isDeleted: true });
 
-    if (response.version_conflicts > 0 && alertIds.length === 1) {
+    if (response.version_conflicts && alertIds.length === 1) {
```

Comment:
won't this not run if `response.version_conflicts === 0` since thats a falsy value?

Reply:
Correct, a value of `0` never satisfied this condition.

Reply:
I wasn't sure if you wanted it to now given the defaults below (should have added that point). 👍🏾

Reply:
Ah, I see! I think `0` makes sense as the default here, I can't think of a more appropriate one.